### PR TITLE
Added ATCache, GoFiles, GoStats and FrisAI.

### PR DESCRIPTION
**Pull request by OGFris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +37,12 @@ A curated list of awesome things and projects built by Algerian developpers.
   - [Rungen](https://github.com/youknowriad/rungen) - A generic runtime around async flow in javascript
 - PHP
   - [Phyber](https://github.com/ghousseyn/phiber) - Build light-weight lightning-fast PhP applications with Phiber
+  - [FrisAI](https://github.com/OGFris/FrisAI) - FrisAI is an A.I. based chat-bot made with PHP.
```
Comment (youknowriad) [inline README.md:39]: Would you mind alphabetically ordering these?

**Review by youknowriad — COMMENTED** [review]
Thanks for the addition :)

File: README.md (hunk 1 above)
Comment (OGFris) [reply README.md:39]: yea my bad

**Review by youknowriad — APPROVED** [review]
👍

**Pull request merged**